tasties_app/tests/test_recipes_view.py

```diff
@@ -22,3 +24,17 @@ def test_view_recipe_unauthorized(self, client):
         response = client.get('/view_recipe/')
         assert response.status_code == 302
         assert response.url == '/login/?next=/view_recipe/'
+
+    def test_add_comment_function(self, client, create_comment):
+        client.post('/login/', data={'username': VALID_USER, 'password': VALID_PASSWORD})
+        response = client.get(f'/view_recipe/?comment-adding={create_comment.comment_text}')
+        assert response.status_code == 302
+        comment_test = Comment.objects.get(pk=create_comment.id)
+        assert comment_test.comment_text == create_comment.comment_text
```

Comment:
@GalOved after you will rebase this pr, you can test similar to what you did here, but instead of "get" make "post" and change the URL to include a certain recipe ID, then validate that the comment for this recipe is saved onto the database.